pkg/v1/cluster/requests_opts.go

```diff
@@ -48,7 +48,7 @@ type CreateOpts struct {
 	Zonal *bool `json:"zonal,omitempty"`
 
 	// KubernetesOptions represents additional k8s options such as pod security policy,
-	// feature gates (Alpha stage only) and admission controllers.
+	// feature gates (Alpha stage only), admission controllers and audit logs.
```

Comment:
Do we have a feature gates in alpha stage? Maybe it's outdated

Reply:
I have found feature gate APIResponseCompression that can be turned on in cluster with version 1.20, but it is in Beta stage since 1.16